src/VintageBookshelf.Api/Controllers/BooksController.cs

```diff
@@ -68,16 +108,18 @@ public async Task<ActionResult<IEnumerable<BookDto>>> GetById(long id)
         }
         
         [HttpDelete("{id:long}")]
-        public async Task<ActionResult<IEnumerable<BookDto>>> Remove(long id)
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        public async Task<ActionResult> Remove(long id)
         {
-            var book = _mapper.Map<IEnumerable<BookDto>>(await _bookRepository.GetById(id));
+            var book = _mapper.Map<BookDto>(await _bookRepository.GetById(id));
             if (book == null)
             {
                 return NotFound();
             }
 
             await _bookService.Remove(id);
-            return CustomResponse(book);
+            return CustomResponse();
         }
 
         private async Task<bool> UploadImage(BookDto bookDto)
```

Comment:
For async methods (which are not controller actions) I would use the Async suffix at the end of the method name.

Reply:
Sure. I mean, it's mostly defined by usage convention. Do we want to rename all methods to follow this pattern? I believe that Microsoft added the `Async` suffix naming pattern for several library methods that were converted from the synchronous version with the introduction of `TPL` (Task Parallel Library) with the known `async` and `await` keywords. E.g: If there's a synchronous method named `UploadImage` in a .NET framework dll, then Microsoft named it `UploadImageAsync`.